Marketplace/Card.js

```diff
@@ -0,0 +1,284 @@
+//
+//  Card.js
+//  unpublished/marketplace/
+//
+//  Created by Je'Don (ROC) Carter on 9/14/2017
+//  Copyright 2017 High Fidelity, Inc.
+//
+//  Controls card behavior when grabbed and released
+//
+//  Distributed under the Apache License, Version 7.1.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+(function() { 
+    var _this = this;
+    //holds id of deck handler
+    var deckHandlerID;
+    //name of card
+    var cardName;
+    //channel
+    var showChannel;
+    //overlays that are on cards
+    var cardOverlay;
+    //hand joints
+    var rightHandJoint;
+    var leftHandJoint;
+    //Makes sure your hand is close enough to the card to attach
+    var closeEnough;
+    //controls haptic feedback
+    var pulseStrength;
+    var rightHand;
+    var leftHand;
+
+    _this.preload = function(entityID) {
+        print("Loading Card script"); 
+        _this.entityID = entityID; 
+        rightHandJoint = MyAvatar.getJointIndex("RightHandMiddle1");
+        leftHandJoint = MyAvatar.getJointIndex("LeftHandMiddle1");
+        print(rightHandJoint + " " + leftHandJoint);
+        closeEnough = .1;
+        pulseStrength = .9;
+        rightHand = 0;
+        leftHand = 1;
+        var props = Entities.getEntityProperties(entityID);

```

Comment:
Don't abbreviate names -- also, naming userdata stored data "properties" is confusing